packages/orbit-components/src/NavigationBar/index.tsx

```diff
@@ -101,8 +57,17 @@ const NavigationBar = ({
   });
 
   return (
-    <StyledNavigationBar data-test={dataTest} id={id} shown={shown}>
-      <StyledNavigationBarContent>{children}</StyledNavigationBarContent>
+    <nav
+      data-test={dataTest}
+      id={id}
+      className={cx(
+        "bg-white-normal shadow-fixed p-sm z-navigation-bar fixed left-0 right-0 top-0 box-border flex w-full translate-x-0 items-center",
+        "duration-normal transition-transform ease-in-out",
+        "tb:h-[64px] h-[52px]", // As defined on the const above
+        shown ? "translate-y-0" : "tb:translate-y-[-64px] translate-y-[-52px]", // As defined on the const above
```

Comment:
Wondering if these should be tokens 🤔 We could use them for the calculations above as well 💭

Reply:
I am okay with having it like you did, not everything should be a token